crates/ludtwig/src/check/rule.rs

```diff
@@ -150,13 +159,27 @@ pub struct CheckNote {
     pub message: String,
 }
 
+#[derive(Debug)]
+pub enum SecondaryNote {
```

Comment:
Not quite sure if this a the best solution, but it gets the job done :thinking:

Reply:
I would rather prefer another member in the check result struct like so
```rust
#[derive(Debug)]
pub struct CheckResult {
    // file_id
    pub(super) rule_name: &'static str,
    pub(super) severity: Severity,
    pub(super) message: String,
    pub(super) primary: Option<CheckNote>,
    pub(super) secondary: Vec<CheckNote>,
    pub(super) suggestions: Vec<CheckSuggestion>,
}
```
Also the `CheckHint` struct is basically the same as the already existing `CheckNote` struct 😉 

But no worries, I can also adjust this if I find the time somewhere in the future 😅